### ISSUE TITLE
ASCII garbage like "11;rgb:0000/0000/0000;1R" printed when running `fig settings`

### ISSUE THREAD

**Issue by mschrage** [withfig/fig#806]
This is quite reproducible.
<img width="443" alt="Screen Shot 2022-01-19 at 9 36 48 PM" src="https://user-images.githubusercontent.com/3155109/150279884-29a7e125-016c-4ab1-bce8-7ccd88e74293.png">

**Comment by webdevnerdstuff** [comment]
This is also happening for me as well. 

I do have my own custom PS1 in my bash profile. I do have a newer version that works with ZSH (I used that one in a linux box so can't test it atm), but generally this one has been good for me for years now. Adding it here in case it helps with the problem solving.

```
function echonewline() {
  echo -e "\n "
}

export PS1=' $(git branch &>/dev/null; if [ $? -eq 0 ]; then \
echo "\[\033[0;36m\]\@ \[\e[0;32m\]\[\033[0;32m\]\u \[\033[0;33m\]\w \[\e[1;35m\]$(git branch | grep ^*|sed s/\*\ //)\
$(echo `git status` | grep "nothing to commit" > /dev/null 2>&1; if [ "$?" -eq "0" ]; then \
echo "\[\e[0;32m\] (clean)"; else \
echo "\[\e[1;31m\] (dirty)"; fi)\[\e[0;32m\] "; else \
echo "\[\033[0;36m\]\@ \[\033[0;32m\]\u \[\033[0;33m\]\w "; fi) \[\e[0m\] \[\033[00m\]$(echonewline) $ \[\033[00m\]'```

**Comment by grant0417** [comment]
This will be fixed in 1.0.56
